### DISCUSSION TITLE
smaller the drone

### DISCUSSION THREAD

**Discussion by Kevinlibaba** [microsoft/AirSim#3910] [Support Q&A]
### What's your question?

Hi all, how can I make the drone smaller in both UE4 and ubuntu.( physical size and Pawn)?

**Comment by jonyMarino** [comment]
Hi @Kevinlibaba and welcome! I think your question can be answered seeing this video:
https://youtu.be/Bp86WiLUC80
See that there is a scale parameter inside UE4 editor.

**Reply by ch-MEIJIE** [reply]
Hi, I want to make sure if I follow the method mentioned in the video and change the scale paremters in UE4(actually I found the paramters corresponding to the default Quadrotor1),  will it also change the physical (dynamics) model of drone? If not, where should I modified the drone model? 
Thank you!

**Reply by ch-MEIJIE** [reply]
I wanna to make myself more claer, for example, I want to control a drone passing through a ring with specific radiu like 60cm, however I found the size of quadcopter from BP_FlyingPawn is almost 1m-1m, it can't pass the ring clearly. If I re-scale the quadcopter to 0.2 (x=0.2, y=0.2, z=0.2). It means that the  quadopter becomes 20cm-20cm from the top view, and it may pass through the ring. However, I actually only changed the size in unreal and I guess no physical paramters were changed, dose it will work by doing so?

**Reply by jonyMarino** [reply]
Hi @ch-MEIJIE. You are right. That doesn't change the physical parameters. For that, you need to change the values at https://github.com/microsoft/AirSim/blob/master/AirLib/include/vehicles/multirotor/MultiRotorParams.hpp

**Comment by Kevinlibaba** [comment]
@jonyMarino  Awesome with my appreciate !! But I also got another problem that how can I modify the Pawn in a released environment from [https://github.com/microsoft/AirSim/releases/](url). e.g.  AirSimNH

**Reply by jonyMarino** [reply]
You can modify the mesh through [these settings](https://microsoft.github.io/AirSim/settings/#pawnpaths). But you won't be able to change the physics of the drone.
